app/helpers/helpers.py

```diff
@@ -0,0 +1,36 @@
+from flask import abort, make_response
+import requests
+from app import token
+
+
+def validate_model(cls, model_id):
+    try:
+        model_id = int(model_id)
+    except:
+        message = f"{cls.__name__} {model_id} is invalid"
+        abort(make_response({"message" : message}, 400))
+    
+    model = cls.query.get(model_id)
+    
+    if not model:
+        message = f"{cls.__name__} {model_id} not found"
+        abort(make_response({"message": message}, 404))
+    
+    return model
+
+def send_message(task):
+
+    url = "https://slack.com/api/chat.postMessage"
```

Comment:
Nice work using a variable to reference the Slack URL. Since it's a constant variable it should be named with all caps